google/cloud/pubsublite/internal/stream_factory.h

```diff
@@ -0,0 +1,85 @@
+#include "absl/container/flat_hash_map.h"
+#include <google/cloud/pubsublite/v1/publisher.grpc.pb.h>
+
+namespace google {
+namespace cloud {
+namespace pubsublite_internal {
+
+template <class Request, class Response>
+using BidiStream = google::cloud::internal::AsyncStreamingReadWriteRpc<Request, Response>;
+
+template <class Request, class Response>
+using StreamFactory = std::function<std::unique_ptr<BidiStream<Request, Response>>()>;
+
+using ClientMetadata = absl::flat_hash_map<std::string, std::string>;
```

Comment:
s/absl::flat_hash_map/std::unordered_map/ the Abseil hash maps have problems with dynamically loaded libraries (possibly these problems are exaggerated, but we are staying away from them for now).

Reply:
Done.